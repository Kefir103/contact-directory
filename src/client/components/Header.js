import React from 'react';

export default function Header() {
    return (<header><h1>Контактный справочник</h1></header>)
}
