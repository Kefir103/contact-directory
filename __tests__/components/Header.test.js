import React from 'react';
import Header from '../../src/client/components/Header';
import { render } from 'enzyme';

describe('<Header />', () => {
    test('should render correctly', () => {
        const component = render(<Header />);
        expect(component).toMatchSnapshot();
    });
});
