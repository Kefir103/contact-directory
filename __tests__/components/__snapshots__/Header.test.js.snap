// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render correctly 1`] = `
<header>
  <h1>
    Пушкарев Е.В. | Тестовое задание для Junior Frontend Developer
  </h1>
</header>
`;
